xml/vk.xml

```diff
@@ -13668,5 +13668,11 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="&quot;VK_NV_extension_346&quot;"       name="VK_NV_EXTENSION_346_EXTENSION_NAME"/>
             </require>
         </extension>
+        <extension name="VK_EXT_extension_347" number="347" author="EXT" contact="Nicolas Caramelli @caramelli" supported="disabled">
```

Comment:
Having accepted the reservation PR, this is the point at which to add the actual XML description of the APIs that was in the original PR.

Reply:
OK, I added the missing section "Valid usage" and I updated the XML description as it was in the original PR https://github.com/KhronosGroup/Vulkan-Docs/pull/1292
Modifications are available on https://github.com/caramelli/Vulkan-Docs

Nicolas Caramelli